### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by jonathan-r-thorpe**
Added normative language disclaimer (which itself uses a normative keyword - how meta!)

Removed other normative keywords

### CODE REVIEW

**Comment by garethsb** [comment]
I would prefer to see something in the Overview instead.

The specs have this:

> ## Use of Normative Language
> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.

So how about, after the **Scope** section in the implementation guide Overview, something like:

> ## Use of Normative Language
> This document is a guide to the AMWA NMOS security specifications. It is not a specification itself, it provides informational guidance around the normative requirements of the relevant specifications. Any normative language key words found in this document are not to be interpreted as RFC 2119 key words.

It's hard to phrase!

**Comment by jonathan-r-thorpe** [comment]
I feel that it is worth mentioning in both places.

The Overview already has this caveat at the end:

> ### Specification Precedence
> This guide is not a substitute for the NMOS Specifications. If there are any inconsistencies between this guide and the specifications, please assume that it is the specification that is correct.
> 

I suggest we replace that section with the following, and move it so it's after Scope. But keep the declaimer in the README.md.

> ### Use of Normative Language
>This document is a guide to the AMWA NMOS security specifications only and not a specification itself. 
>It provides informational guidance around the normative requirements of the relevant specifications but any normative language key words found in this document are not to be interpreted as RFC 2119 key words. 
>If there are any inconsistencies between this guide and the specifications, please assume that it is the specification that is correct.

WDYT?

File: README.md [hunk 1]
```diff
@@ -10,15 +10,16 @@ This repository holds the source for this Specification, part of the family of [
 ### What does it do?
 
 - It's a guide for implementers who want to add NMOS Security to their NMOS Nodes and/or NMOS Controllers.
+- Please note that this is NOT a specification and therefore none of its contents should be interpreted as normative.
```
Comment (garethsb) [inline README.md]:
```suggestion
- Please note that this guide is informative and therefore is not a substitute for reading the normative specifications.
```

**Comment by prince-chrismc** [comment]
> This repository holds the source for this Specification

Can you please tweak the opening sentence for the README as well?

File: README.md [hunk 2]
```diff
@@ -3,22 +3,23 @@
 [![Lint Status](https://github.com/AMWA-TV/nmos-template/workflows/Lint/badge.svg)](https://github.com/AMWA-TV/nmos-template/actions?query=workflow%3ALint)
 [![Render Status](https://github.com/AMWA-TV/nmos-template/workflows/Render/badge.svg)](https://github.com/AMWA-TV/nmos-template/actions?query=workflow%3ARender)
 
-This repository holds the source for this Specification, part of the family of [Networked Media Open Specifications](https://specs.amwa.tv/nmos) from the [Advanced Media Workflow Association](https://amwa.tv)
+This repository holds the source for this Implementation Guide, part of the family of [Networked Media Open Specifications](https://specs.amwa.tv/nmos) from the [Advanced Media Workflow Association](https://amwa.tv)
```
Comment (jonathan-r-thorpe) [inline README.md:6]: >part of the family of Networked Media Open Specifications

Kind of suggests its a spec.  Maybe something like:

>This repository holds the source for this Implementation Guide, _which is a guide to_ the Networked Media Open Specifications from the Advanced Media Workflow Association

Comment (garethsb) [reply README.md:6]: Wayward nephew? Touchy-feely uncle?

Comment (jonathan-r-thorpe) [reply README.md:6]: Probably best to leave as 'part of the family' then :)

File: README.md [hunk 3]
```diff
@@ -3,22 +3,23 @@
...
 ### Why does it matter?
 
-- This implementation guide should make the job of adding NMOS Security to your implementations easier.
+- This implementation guide makes the job of adding NMOS Security to your implementations easier.
 
 ### How does it work?
 
 - It details what you need to know, and the steps you need to take when implementing security.
-- It includes some recommended tools to help in your implementation
+- It also recommends some tools to help in your implementation
```
Comment (garethsb) [inline README.md]:
```suggestion
- It also recommends some tools to help in your implementation.
```

**Review by garethsb — APPROVED** [review]

**Pull request merged**